### PR TITLE
Update for MDVA-41061 in QPT 1.1.15

### PR DESCRIPTION
**Pull request by katmoon**
- Corrected previously incorrect labels, added a new supported version and information about the update in QPT 1.1.15

### CODE REVIEW

**Comment by Rinn22** [comment]
@magento import code to magento-kb/knowledge-base

**Comment by m2-github-services** [comment]
@Rinn22 the branch with the code is successfully imported into the `magento-kb/knowledge-base` repository. Branch name: `imported-katmoon-knowledge-base-144`.

File: src/support-tools/patches-available-in-qpt-tool/mdva-41061-stock-status-resets-saleable-when-saved-from-admin.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+labels: Support Tools,QPT patches,Quality Patches Tool,Magneto Commerce Cloud,QPT 1.1.5,QPT 1.1.15,Adobe Commerce,cloud infrastructure,on-premises,stock status,saleable,2.4.2,2.4.2-p1,2.4.2-p2,2.4.3,2.4.3-p1,2.4.3-p2
 ---
 
-The MDVA-41061 patch fixes the issue where the stock status resets to saleable when the product is saved from the Admin. This patch is available when the [Quality Patches Tool (QPT)](https://devdocs.magento.com/guides/v2.4/comp-mgr/patching.html#mqp) 1.1.5 is installed. The patch ID is MDVA-41061. Please note that the issue is scheduled to be fixed in Adobe Commerce 2.4.4.
+The MDVA-41061 patch fixes the issue where the stock status resets to saleable when the product is saved from the Admin. This patch is available when the [Quality Patches Tool (QPT)](https://devdocs.magento.com/guides/v2.4/comp-mgr/patching.html#mqp) 1.1.5 is installed. The latest patch version is available in QPT 1.1.15. The patch ID is MDVA-41061. Please note that the issue is scheduled to be fixed in Adobe Commerce 2.4.4.
```
Comment (Rinn22) [inline src/support-tools/patches-available-in-qpt-tool/mdva-41061-stock-status-resets-saleable-when-saved-from-admin.md:6]: @katmoon What is the Patch ID in 1.1.15?

Comment (Rinn22) [inline src/support-tools/patches-available-in-qpt-tool/mdva-41061-stock-status-resets-saleable-when-saved-from-admin.md:6]:
```suggestion
The MDVA-41061 patch fixes the issue where the stock status resets to saleable when the product is saved from the Admin. This patch is available when the [Quality Patches Tool (QPT)](https://devdocs.magento.com/guides/v2.4/comp-mgr/patching.html#mqp) 1.1.5 is installed. The patch ID is MDVA-41061. The latest patch version is available in QPT 1.1.15.  Please note that the issue is scheduled to be fixed in Adobe Commerce 2.4.4.
```

Comment (Rinn22) [reply src/support-tools/patches-available-in-qpt-tool/mdva-41061-stock-status-resets-saleable-when-saved-from-admin.md:6]: @katmoon I am not very familiar with how the patch update works. Should we include the patch ID in the article and say something like: **"The latest patch version is available in QPT 1.1.15 as patch ID - MDVA-41061-V3."** ?

Also why MDVA-41061-V3 is not included in this ticket?: https://jira.corp.adobe.com/browse/MSKB-1580

**Review by katmoon — COMMENTED** [review]
Looks good

**Comment by Rinn22** [comment]
@magento import code to magento-kb/knowledge-base

**Comment by m2-github-services** [comment]
@Rinn22 the branch with the code is successfully imported into the `magento-kb/knowledge-base` repository. Branch name: `imported-katmoon-knowledge-base-144`.

**Pull request merged**